tests/test_unit.py

```diff
@@ -120,7 +120,7 @@ def test_reverse_multiply():
 def test_sub():
     C = ansunits.Unit("C")
     kg = ansunits.Unit("kg")
-    delta_C = C - C
+    delta_C, _ = C - C
```

Comment:
@ansaboutin What's the second part?

Reply:
Here it would be the absolute temperature counterpart. This works well for conversions to and from relative units but I am unsure if there would be a cleaner way.